source/extensions/filters/http/aws_lambda/config.cc

```diff
@@ -137,10 +137,12 @@ void AWSLambdaConfigImpl::init(Event::Dispatcher &dispatcher) {
         try {
             const auto web_token = shared_this->api_.fileSystem().fileReadToEnd(
                 shared_this->token_file_);
-            shared_this->tls_.runOnAllThreads(
-                [web_token](OptRef<ThreadLocalCredentials> prev_config) {
-                  prev_config->sts_credentials_->setWebToken(web_token);
-                });
+            if (web_token != "") {
```

Comment:
If this is empty, should we reject? Is it even worth starting up without this data? Or can it fill in later?

Reply:
Line 123 we check that. This is technically in our timer so this to reject invalid updates (wont fix everything but should help avoid some bad updates). We could also increment a stat if we wanted.

Reply:
Ahhh I see what you mean, yea I think definitely a stat